scheduler/prioritize_test.go

```diff
@@ -16,22 +17,24 @@ func TestCapacityToScore(t *testing.T) {
 		expect  int
 	}{
 		{0, 1, 0},
-		{1, 1, 0},
+		{1, 1, 1}, // even one byte will lead to a score of at least 1
 		{128 << 30, 1, 7},
 		{128 << 30, 2, 6},
 		{128 << 30, 0.5, 8},
 		{^uint64(0), 1, 10},
```

Comment:
Could you add a test for the case where `converted` is less than 1?
https://github.com/topolvm/topolvm/pull/766/files#diff-ffe695214ecf8679e2ecd8e76e947ced21ad9539fe64f32f116aeb181f7064cfR32-R33

Reply:
I added the case
```
{1 << 30, 1, 1}, // converted < 1 but still gets a score of 1 because of the minimum
```
for this. Behaves as expected now.